prusti-viper/src/encoder/encoder.rs

```diff
@@ -794,9 +753,12 @@ impl<'v, 'tcx> Encoder<'v, 'tcx> {
                 },
             };
             self.log_vir_program_before_viper(method.to_string());
+            // TODO: ???
+            /*
```

Comment:
This is the call that performs the actual optimization. Why is it commented out?